tests/python/pants_test/pantsd/pantsd_integration_test.py

```diff
@@ -57,6 +57,7 @@ def join():
 }
 
 
+@pytest.mark.skip(reason="flaky test, see #18664")
```

Comment:
I'm pretty sure that you don't need to skip all of these: there are two in particular that are flaky, IME:
* `test_pantsd_invalidation_pants_toml_file`
* `test_pantsd_invalidation_file_tracking`